hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/TokenClient.java

```diff
@@ -650,6 +651,40 @@ public NetworkTransactionResponse updateTokenFeeSchedule(
         return response;
     }
 
+    public NetworkTransactionResponse rejectFungibleToken(List<TokenId> tokenIds, AccountId ownerId, PrivateKey privateKey, ExpandedAccountId payer) {
```

Comment:
If owner is set, which we do, then it must sign the transaction. Carol is also paying for the transaction so there's no need to pass 3 variables for carol when a single will suffice.

```suggestion
    public NetworkTransactionResponse rejectFungibleToken(List<TokenId> tokenIds, ExpandedAccountId owner) {
```

Reply:
fixed